### DISCUSSION TITLE
Will Kong GUI be available in open source ?

### DISCUSSION THREAD

**Discussion by junevm** [Kong/kong#8661] [Help] [answered]
Will Kong GUI be made available in open source edition in future?

**Comment by mayocream** [comment, marked as answer]
The Kong Manager is available in Free Mode; we don't have a plan to make an open-source dashboard now.

[Installation paths](https://docs.konghq.com/gateway/2.8.x/install-and-run/): 
> Free Mode: Install Kong Gateway without a license, gaining access to Kong Manager.
